models/print_printer.py

```diff
@@ -67,33 +71,42 @@ def _spool_lpr(self, document, title=None):
                                 (str(lpr.returncode), output))
 
     @api.multi
-    def spool(self, document, title=None):
+    def spool(self, document, title=None, copies=None):
         """Spool document to printer"""
         # Spool document via OS-dependent spooler mechanism
         if os.name == 'posix':
-            self._spool_lpr(document, title)
+            self._spool_lpr(document, title, copies)
         else:
             raise UserError(_('Cannot print on OS: %s' % os.name))
         return True
 
     @api.multi
-    def spool_report(self, docids, report_name, html=None, data=None,
-                     title=None):
+    def spool_report(self, docids, report_name, data=None, title=None,
+                     copies=None):
         """Spool report to printer"""
         # Generate PDF report
-        document = self.env['report'].get_pdf(docids, report_name, html, data)
+        report = self.env['ir.actions.report'].search(
+                 [('report_name', '=', report_name)])
+
+        if not report:
+            raise UserError(_('Cannot find report: %s' % report_name))
+
+        # render_qweb_pdf returns a tuple of (content,'pdf')
+        document = report.render_qweb_pdf(docids, data)[0]
+
         # Use report name and document IDs as title if no title specified
         if title is None:
             title = ('%s %s' % (report_name, str(docids)))
+
         # Spool generated PDF to printer(s)
-        self.spool(document, title=title)
+        self.spool(document, title=title, copies=copies)
         return True
 
     @api.multi
     def spool_test_page(self):
         """Print test page"""
         for printer in self._printers():
-            printer.spool_report(printer.ids, 'print.report_test_page',
+            printer.spool_report(printer.ids, 'odoo-print.report_test_page',
```

Comment:
We are renaming module back to print, so odoo-print must be changed back to print.